### ISSUE TITLE
Let status messages be set by elevated users

### ISSUE THREAD

**Issue by cl8n** [RoddersGH/DekuBot#22]
- Faction welcome message
- Many others... 😉

**Comment by Kriscotiano** [comment]
I absolutely agree with this because so far the faction pm to a user is dismissed by many in my server because they think it just lists information and never realize it has factions they may choose from. Maybe if I could move that info to the help command it would be much more appropriate.

**Comment by cl8n** [comment]
Yep, pretty much anything DekuBot says should (and will) be customize-able.
